src/rest/mormot.rest.client.pas

```diff
@@ -748,6 +748,9 @@   TRestClientUri = class(TRest)
     function ServiceDefineSharedApi(const aInterface: TGUID;
       const aContractExpected: RawUtf8 = SERVICE_CONTRACT_NONE_EXPECTED;
       aIgnoreAnyException: boolean = false): TServiceFactoryClient;
+    // undefine previously defined Interfaces

```

Comment:
please use 3 / for the first comment line ///
(for proper documentation generation)

Reply:
Done.